### DISCUSSION TITLE
Pennywise: NukeSped malware

### DISCUSSION THREAD

**Discussion by donkeykong74** [Homebrew/discussions#3341] [Casks] [answered]
Hi, I just installed pennywise on my MacOS, CleanMyMac warned my about malware while installing. 
Is this an error or true malware in the pennywise brew package pennywise--0.8.0.dmg 

Thanks!

**Comment by SMillerDev** [comment, marked as answer]
You'll have to ask the developer here: https://github.com/kamranahmedse/pennywise

**Reply by donkeykong74** [reply]
Thanks, will do!
